emcc.py

```diff
@@ -1691,13 +1691,12 @@ def default_setting(name, new_default):
         '__heap_base',
         '__stack_pointer',
     ]
-    settings.EXPORTED_FUNCTIONS += [
-        # This needs to be exported on the Module object too so it's visible
-        # to side modules too.
-        '___heap_base',
-        # Unconditional dependency in library_dylink.js
-        '_setThrew',
-    ]
+    # This needs to be exported on the Module object too so it's visible
+    # to side modules too.
+    settings.EXPORTED_FUNCTIONS.append('___heap_base')
```

Comment:
why does this not use REQUIRED_SYMBOLS as well?

Reply:
When `RELOCATABLE` code `__heap_base` (like `__stack_pointer`) is defined, in JS code, not in native code, so it can't go in the `REQUIRED_SYMBOLS`.